### DISCUSSION TITLE
Data Table Theme

### DISCUSSION THREAD

**Discussion by aloiseau17** [vuetifyjs/vuetify#11493] [General]
Hi,

I can't apply dark theme on pagination of `<v-data-table>` on this codepen:
[https://codepen.io/Auremy/pen/dyYEPVm](https://codepen.io/Auremy/pen/dyYEPVm)

Does someone have an idea ?

**Comment by aloiseau17** [comment]
It's pagination menu opened that is not dark
`<v-menu>` as `<v-theme-provider>` to light.

![image](https://user-images.githubusercontent.com/19550407/82922598-15df3e80-9f7a-11ea-94e8-66619e2423e9.png)

**Comment by johnleider** [comment]
While it is standard for v-menu to use the root status of **dark/light**, I think that an explicit value should propagate from `v-data-table` down. @nekosaur thoughts?

**Reply by nekosaur** [reply]
Yes it should

**Comment by aloiseau17** [comment]
So is it consider as a bug ?
